src/utils.ts

```diff
@@ -112,7 +115,7 @@ export async function getDeletedFiles({
     }
   }
 
-  for (const filePath of await deletedGitFiles({baseSha, sha, cwd})) {
+  for (const filePath of await deletedGitFiles({baseSha, sha, cwd, diff})) {
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'diff' is not defined.